content/applications/websites/website/structure/seo.rst

```diff
@@ -191,34 +234,42 @@ methods:
    - :doc:`../configuration/google_search_console`
    - :doc:`pages`
 
-Sitemap
--------
+Prevent the website of a test database from appearing in search results
+-----------------------------------------------------------------------
 
-The sitemap points out website pages and their relation to each other to search engine robots. Odoo
-generates a `/sitemap.xml` file, including all URLs. For performance reasons, this file is cached
-and updated every 12 hours.
+To prevent a test database website from appearing in search engine results, add any placeholder
+value in the domain field. Go to :menuselection:`Configuration --> Settings`, then in
+the :guilabel:`Website Info` section, add a random value in the :guilabel:`Domain` field. Doing so
+automatically inserts the following tag into the page source:
```

Comment:
```suggestion
To prevent a website from appearing in search engine results, go to :menuselection:`Website --> Configuration --> Settings`, then in
the :guilabel:`Website Info` section, add a random value in the :guilabel:`Domain` field. Doing so
automatically inserts the following tag into the page source:
```